src/api/app/controllers/webui/patchinfo_controller.rb

```diff
@@ -185,46 +182,42 @@ def new_tracker
     invalid_format = ''
     # params[:issues] = list of new issues to add
     params[:issues].each do |new_issue|
-      # issue = collecting all informations of an new issue
-      issue = []
-      if new_issue.starts_with? 'CVE-'
-        issue[0] = 'cve'
-        issue[1] = new_issue
-      else
-        issue = new_issue.split('#')
-      end
-      if issue.length > 1
+      issue = IssueTracker::IssueTrackerHelper.new new_issue
```

Comment:
We should have a cop for this, please and the missing parenthesis of the method `IssueTracker::IssueTrackerHelper.new(new_issue)`

Reply:
there is no missing parenthesis, @DavidKang. If you want to enforce a specific style, please let's do it via rubocop.

Reply:
There is already a cop for that and it's super fun :)

1058 files inspected, 17732 offenses detected, 17726 offenses corrected